Resources/Prototypes/_NF/Roles/Jobs/Pirates/pirate.yml

```
@@ -5,7 +5,7 @@
   playTimeTracker: JobPirate
   requirements:
     - !type:OverallPlaytimeRequirement
-      time: 10800 # 3 hrs - mono
+      time: 54000 # 15 hrs - mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)